contracts/cryptoblades.sol

```diff
@@ -998,4 +1020,28 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
         return _getRewardsClaimTax(msg.sender);
     }
 
+    function getMintWeaponFee() public view returns (int128) {
+        int128 decrease = ABDKMath64x64.divu(block.timestamp.sub(vars[VAR_WEAPON_MINT_TIMESTAMP]).mul(vars[VAR_MINT_FEE_DECREASE_SPEED]), 1e18);
+        int128 weaponFeeMin = ABDKMath64x64.divu(vars[VAR_MIN_WEAPON_FEE], 100);
```

Comment:
Maybe worth setting in wei and dividing by 100*10e18 to allow for more precise floor price updates (same for VAR_MIN_CHARACTER_FEE).